eng/pipelines/fluent_integration.yaml

```diff
@@ -42,7 +42,7 @@ jobs:
       jdkArchitectureOption: 'x64'
       publishJUnitResults: true
       testResultsFiles: '**/TEST-*.xml'
-      goals: 'verify'
+      goals: 'clean install'
```

Comment:
Thanks. Previous fluent does not depends on azure-autorest-parent hence it does not need a installed lib of it. Now it does.